docs/organization/export/README.md

```diff
@@ -0,0 +1,39 @@
+# Contentful CLI - `organization export` command
+
+This command helps you export organization data to a json file
+
+## Usage
+
+```
+contentful organization export
+
+export a space data to a json file
```

Comment:
```suggestion
export an organization's data to a json file
```

could even be more specific that it is just taxonomy data rn to cut down on confusion, up to you